### DISCUSSION TITLE
Sequentially request in loop

### DISCUSSION THREAD

**Discussion by darkeum** [reactphp/reactphp#552] [Q&A]
I have a task to async poll several servers via a socket, but one server must receive several requests in turn (servers only allow one connection at a time). How to do it?

**Comment by SimonFrings** [comment]
@darkeum Thanks for bringing this up :+1:

I'm having a hard time understanding what exactly you're looking for. Can you explain your use case a bit further, what you're trying to achieve and how you're currently blocked in doing so?
